openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -843,7 +843,7 @@ def enumerate_protomers(
         options.SetMaxCount(max_states)
 
         return [
-            self.from_openeye(
+            self.from_openeye(  # type: ignore[misc]
```

Comment:
I swear this caused me issues when I tried it, but I gave it a spin now and no issues. (Maybe there are differences between Python and mypy versions since I last tried, I'm not going to dig unless it's needed.)